### PR TITLE
Add Nolali

### PR DESCRIPTION
**Pull request by Spandamn**

### CODE REVIEW

File: data/mods/ssb/abilities.ts [hunk 1]
```diff
@@ -662,6 +662,32 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
...
 	},
 
+	// Nolali
+	burningsoul: {
+		shortDesc: "Drought + Sturdy.",
+		onStart(source) {
+			for (const action of this.queue) {
+				if (action.choice === 'runPrimal' && action.pokemon === source && source.species.id === 'groudon') return;
+				if (action.choice !== 'runSwitch' && action.choice !== 'runPrimal') break;
+			}
```
Comment (ghost) [inline data/mods/ssb/abilities.ts]: this thing cant go primal, and theres no groudon. dw bout this line

Comment (ghost) [reply data/mods/ssb/abilities.ts]: someone smack me if someone decides to go skill swap red orb groudon for whatever reason

**Review by ghost — APPROVED** [review]
Looks good

**Comment by Spandamn** [comment]
done

**Pull request merged**